packages/utilities/psammead-storybook-helpers/src/text-variants.js

```diff
@@ -1,71 +1,175 @@
-  ChineseTrad: '家長們在學校門口維權。',
-  English: 'Kepler telescope discovers 100 Earth-sized planets',
-  Gahuza:
-    'Abashika 100.000 ntawuzi irengero ryabo, abandi ibihumbi nabo ntibagira aho bakika umusaya',
-  Gujarati: 'જીતેન્દ્રસિંહ મૂળ ઉત્તર પ્રદેશના ફિરોઝાબાદના',
-  Hausa: 'Bayanin Ganduje kan ayyukan Gama',
-  Hindi: 'जय शाह',
-  Igbo:
-    "Trade Fair: Ọ bụrụgodi na gọọmenti etiti chọrọ ire ọdọ ahịa a, ndị Igbo ga-egonwu ya'",
-  Indonesian:
-    '"Apa yang terjadi di Selandia Baru adalah kekerasan yang dibawa oleh seseorang yang tumbuh dan belajar ideologinya di tempat lain,\' kata Jacinda Ardern.',
-  Japanese: '度目の採決認めなかった理由は？ 英下院議長に取材',
-  Korean: '마이크 폼페이오 미국 국무장관',
-  Kyrgyz: 'Казакстан Назарбаевден башка президентти көрө элек',
-  Marathi: 'नीरव मोदी',
-  Mundo:
-    'La caída en los precios de la vivienda ha hecho que algunas personas puedan comprar en zonas acomodadas de la ciudad.',
-  Nepali: 'हेमन्तप्रकाश मल्ल',
-  Pashto:
-    'د ملګرو ملتونو د ماشومانو ادارې یونیسف افغان کرېکټ لوبغاړی راشد خان په افغانستان کې د ښه نیست ملي سفیر وټاکه.',
-  Persian:
-    'در این جشنواره برای نخستین بار از کارگران افغانستان در شهر تهران تقدیر شد',
-  Pidgin:
-    'Di Nigeria Army don set up 9 man Committee to chook eye inside how dia men conduct demsef during di 2019 General Elections.',
-  Punjabi:
-    'ਪਾਕਿਸਤਾਨੀ ਮਹਿਲਾ ਰਾਹਿਲਾ ਨੇ ਭਾਰਤੀ ਵਕੀਲ ਜ਼ਰੀਏ ਅਦਾਲਤ ਵਿੱਚ ਦਿੱਤੀ ਅਰਜ਼ੀ ਵਿੱਚ ਕਿਹਾ ਕਿ ਮਾਮਲੇ ਨਾਲ ਜੁੜੇ ਪਾਕਿਸਤਾਨੀ ਗਵਾਹਾਂ ਨੂੰ ਬੁਲਾਇਆ ਜਾਵੇ',
-  Russian:
-    'Мнение Назарбаева будет иметь приоритетное значение при принятии важных для страны решений, сказал Токаев',
-  SerbianCyr: 'Караџић се годинама крио пре него што је ухапшен 2008. године',
-  SerbianLat: 'Karadžić se godinama krio pre nego što je uhapšen 2008. godine',
-  Sinhala: 'සිවිල් යුද ගැටුම් හමුවේ කොටු වී සිටි සිවිල් වැසියන්',
-  Somali: 'Qaar ka mid ah dadkii laga soo badbaadiyay Hotel Dusit2 ee Nairobi',
-  Swahili:
-    'Kimbunga Idai: ‘Watu milioni 1.7 wapo katika njia kuu’ ya kimbunga Msumbiji, Zimbabwe',
-  Tamil: 'நீரவ் மோதி',
-  Telugu:
-    'నల్లగొండ ఫ్లోరైడ్ సమస్య గురించి నాటి ప్రధాని అటల్ బిహారీ వాజపేయికి దుశ్చర్ల సత్యనారాయణ వివరించారు',
-  Thai: 'ภาพวาดของตำรวจจากใบหน้าผู้เสียชีวิต',
-  Tigrinya: 'ዓብዱራሕማን ኣቡሃሽም',
-  Turkish:
-    'Kassım Jomart Tokayev, parlamentoda yemin ederek devlet başkanlığı görevini resmen devraldı.',
-  UKChinaSimp:
-    '该计划的批评者说，这个政策不能解决住房短缺的问题（Credit: Alamy）',
-  UKChinaTrad:
-    '該計劃的批評者說，這個政策不能解決住房短缺的問題（Credit: Alamy）',
-  Ukrainian:
-    "Олег Ляшко був традиційно емоційним, говорив на підвищених тонах. Прем'єр спочатку нервував, проте швидко опанував ситуацію.",
-  Urdu:
-    'وزیر اعظم جاسنڈا آرڈرن کا کہنا ہے کہ حملہ آور کا نام کسی واچ لسٹ پر نہیں تھا، نہ آسٹریلیا میں اور نہ ہی کہیں اور',
-  Uzbek:
-    'Дариға Назарбоева Қозоғистон президентининг Британияга охирги сафарида отасига ҳамроҳлик қилган',
-  Vietnamese: 'Ông Nazarbayev bất ngờ tuyên bố từ chức hôm 19/3/2019',
-  Yoruba: 'Wo àwọn òrílẹ̀ èdè Mẹ́wàá tó láyọ̀ jùlọ Lágbàyé',
+  AfaanOromoo: {
```

Comment:
Ideally these keys shouldn't be capitalised otherwise we end up with syntax like [`LANGUAGE_VARIANTS.English.text;`](https://github.com/bbc/psammead/pull/468/files#diff-c2e7d4722b8df6ba5cc5dfab8c49be40R16) but that could be handled in a follow up PR